packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -300,3 +372,72 @@ export class RouterliciousOrdererRestWrapper extends RouterliciousRestWrapper {
 		return restWrapper;
 	}
 }
+
+export function getW3CData(url: string) {
```

Comment:
Can we put it into shared code such that we can share code between FRS & ODSP driver?